models/zeroex/ethereum/zeroex_ethereum_api_fills.sql

```diff
@@ -311,19 +311,19 @@ direct_uniswapv2 AS (
             swap.contract_address AS maker,
             LAST_VALUE(swap.to) OVER ( PARTITION BY swap.evt_tx_hash ORDER BY swap.evt_index) AS taker,
             CASE
-                WHEN swap.amount0In > swap.amount0Out THEN pair.token0
+                WHEN swap.amount0In::float > swap.amount0Out::float THEN pair.token0
```

Comment:
please use `CAST()` rather than `::` to help prepare for running on dunesql (this applies throughout PR)

Reply:
Thanks for pointing this out.
I have made the necessary changes.